frontstage/views/account/account_survey_share.py

```diff
@@ -23,8 +23,7 @@
 @account_bp.route('/share-surveys', methods=['GET'])
 @jwt_authorization(request)
 def share_survey_overview(session):
-    flask_session.pop('share_survey_business_selected', None)
-    flask_session.pop('share_survey_surveys_selected', None)
+    flask_session.pop('share', None)
```

Comment:
Can this have a better name? A key of 'share' in the session doesn't say a huge amount about what it's for

Reply:
👍